include/pagmo/population.hpp

```diff
@@ -287,6 +288,24 @@ class population
         return worst_idx(tol_vector);
     }
 
+    /// Gets the champion decision vector
+    vector_double get_champion_x() const {
+        if (m_prob.get_nobj() > 1u) {
+            pagmo_throw(std::invalid_argument,
+                        "The Champion of a population can only be extracted in single objective problems");
+        }
+        return m_champion_x;
```

Comment:
Philosophical questions: what are the pros/cons of throwing if the champion has size zero, rather than returning an empty vector? Do we want to be silent if the user erroneously asks for a champion from an empty pop? Should we have a method that checks whether the champion exists (``has_champion()``)?